.buildkite/deploy/kubernetes/deploy-k8s-pipeline.yml

```diff
@@ -46,12 +46,12 @@ steps:
         machineType: "${GCP_HI_PERF_MACHINE_TYPE}"
       notify:
         - github_commit_status:
-            context: "deploy/k8s test v1.30.2"
+            context: "deploy/k8s test v1.31.0"
 
-    - label: "K8S Test/K8S version: v1.29.4"
-      key: "k8s-test-129"
+    - label: "K8S Test/K8S version: v1.30.2"
+      key: "k8s-test-130"
       env:
-        K8S_VERSION: "v1.29.4"
+        K8S_VERSION: "v1.30.2"
```

Comment:
1.30.4 is the latest available I see in https://hub.docker.com/r/kindest/node/tags

Reply:
Thanks, I will add the changes in a single commit.